docs/UserGuide.md

```diff
@@ -11,117 +11,222 @@
    > Having any Java 8 version is not enough. <br>
    This app will not work with earlier versions of Java 8.
    
-1. Download the latest `addressbook.jar` from the 'releases' tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `mastermind.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for your Mastermind.
 3. Double-click the file to start the app. The GUI should appear in a few seconds. 
    > <img src="images/Ui.png" width="600">
 
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
+   * **`list`** : lists all floating tasks, events and tasks due today
+   * **`add`**` CS2103T "tutorial work" by/1100 on/tomorrow t/TUTORIAL` : 
+     adds a task named `CS2103T tutorial work` with deadline due tomorrow at 11am to Mastermind.
+   * **`delete`**` 3` : deletes the 3rd task shown in the current list
    * **`exit`** : exits the app
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
 ## Features
 
-#### Viewing help : `help`
+### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+### Adding a task: `add` `do` `complete`
```

Comment:
you need to specify somewhere, the date and time formats you support in the application.